rdflib/paths.py

```diff
@@ -462,6 +462,17 @@ def path_sequence(self, other):
 
 
 def evalPath(graph, t):
+    warnings.warn(
+        DeprecationWarning(
+            "rdflib.path.evalPath() is deprecated, use the (all-lowercase) evalpath(). "
+            "The mixed-case evalPath() function name is incompatible with PEP8 "
+            "recommendations and will be replaced by evalpath() in rdflib 7.0.0."
+        )
+    )
+    return evalpath(graph, t)
+
+
+def evalpath(graph, t):
```

Comment:
Given the naming of other functions like `mul_path`, `inv_path`, `neg_path` I think it may be better to go with `eval_path` instead.

Happy to change it in your branch if you agree.